packages/react/src/components/CommandBar/CommandBar.test.tsx

```diff
@@ -12,45 +11,30 @@ describe('CommandBar', () => {
     resetIds();
   });
 
-  afterEach(() => {
```

Comment:
not sure why this was needed, but are we sure that removing it wont leave host with lot of churn that might potentially conflict with other tests?

Reply:
Not needed anymore since `testing-library` automatically runs a `cleanup` to unmount the react trees that were rendered after a `render` call in jest